rlclientlib/vw_model/safe_vw.cc

```diff
@@ -192,7 +245,9 @@ bool safe_vw::is_compatible(const std::string& args) const {
   //TODO: proper implentation for more than 2 algorithms. Changes inside VW are required
   const bool is_ccb_init = args.find("ccb_explore_adf") != std::string::npos;
   const bool is_ccb = _vw->options->was_supplied("ccb_explore_adf");
-  return is_ccb_init == is_ccb;
+  const bool is_slates_init = args.find("slates") != std::string::npos;
```

Comment:
isn't "slates" adding "ccb_explore_adf" in setup? In this case we probably wouldn't detect switch from between ccb and slates (since is_ccb_init will be always equal to is_ccb).